packages/block-library/src/quote/v2/save.js

```diff
@@ -1,5 +1,27 @@
-function Save() {
-	return <div>Quote block v2</div>;
-}
+/**
+ * WordPress dependencies
+ */
+import {
+	RichText,
+	useBlockProps,
+	useInnerBlocksProps,
+} from '@wordpress/block-editor';
+
+export default function save( { attributes } ) {
+	const { attribution } = attributes;
+	const wrapperProps = useBlockProps.save();
+	const innerBlocksProps = useInnerBlocksProps.save();
+	const allBlockProps = useInnerBlocksProps.save( wrapperProps );
 
-export default Save;
+	const hasAttribution = ! RichText.isEmpty( attribution );
+	return hasAttribution ? (
+		<figure { ...wrapperProps }>
```

Comment:
The markup here seems different than the existing quote block. Are we ok with this change, I'm expecting some breakage for themes (so dev note and all that stuff)

Reply:
According to this conversation, it seems we are: https://github.com/WordPress/gutenberg/pull/25892#discussion_r589615279

One thing that wasn't implemented there but I read we wanted to do is to use `<blockquote>` or `<figure>` depending on whether the quote has a cite or not.

Reply:
> If I don't have attribution, it doesn't display the innerBlocks in code view and front end.

Fixed.